src/models/propulsion/FGTurbine.cpp

```diff
@@ -234,6 +234,12 @@ double FGTurbine::Run()
   OilPressure_psi = N2 * 0.62;
   OilTemp_degK = Seek(&OilTemp_degK, 366.0, 1.2, 0.1);
 
+  // Check if there are functions for TSFC and ATSFC
+  if (TSFCLookup)
+    TSFC = TSFCLookup->GetValue();
+  if (ATSFCLookup)
+    ATSFC = ATSFCLookup->GetValue();
+
```

Comment:
The point of making `TSFC` and `ATSFC` pointers of type `FGParameter*` is that you no longer need to test if `TSFCLookup` and `ATSFCLookup` exist and just call `GetValue()` on them:
```suggestion
  double _TSFC = TSFC->GetValue();
  double _ATSFC = ATSFC->GetValue();
```